src/Video.tsx

```diff
@@ -606,14 +610,56 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
         : ViewType.SURFACE;
     }, [drm, useSecureView, useTextureView, viewType]);
 
+    const posterStyle = useMemo<StyleProp<ImageStyle>>(() => {
```

Comment:
is it possible to move the content of this memo inside _renderPoster ?
As it is used only in _renderPoster I think it can avoid unnecessary computations